src/main/routes/config.ts

```diff
@@ -5,6 +5,31 @@ import { getAppVersion } from "../../common/app-utils"
 import { CoreV1Api, AuthorizationV1Api } from "@kubernetes/client-node"
 import { Cluster } from "../cluster"
 
+// TODO: auto-populate all resources dynamically
+const apiResources = [
+  { resource: "configmaps" },
```

Comment:
I would keep this resource keys in some `enum` to avoid repeating strings over whole project, 
e.g. `isAllowedResource("cronjobs")`, otherwise LGTM.
Let's merge / refactor later.

Reply:
Or we could make it as normal object with predefined props, e.g. like `accessCheckerObj.pods`

Reply:
I agree that strings are not the best way to go... let's refactor this in follow up PRs.